asn1sccserviceprovider.cpp

```diff
@@ -145,8 +145,9 @@ void Asn1SccServiceProvider::loadServiceParams()
 void Asn1SccServiceProvider::onProcessFinished(int exitCode, QProcess::ExitStatus exitStatus)
 {
     Q_UNUSED(exitCode);
+    Q_UNUSED(exitStatus);
 
-    if (!m_terminating && exitStatus != QProcess::NormalExit) {
+    if (!m_terminating) {
```

Comment:
This is because when I was testing how QProcesses behave when crashing, I did not use asn1scc, but have prepared simple test application which actually crashed. Asn1scc does not crash but exits normally on failure, so this condition does not make sense.